src/main/java/edu/bu/metcs673/trackr/controller/BankAccountController.java

```diff
@@ -87,4 +84,28 @@ public ResponseEntity<GenericApiResponse> modifyBankAccount(@PathVariable(value
                         MessageFormat.format(CommonConstants.DEACTIVATE_BANK_ACCOUNT, String.valueOf(id))),
                 HttpStatus.OK);
     }
+    @GetMapping("/find/{userId}/{bankAccountId}")
+    public ResponseEntity<GenericApiResponse1<BankAccount>> findBankAccountById(@PathVariable(value = "userId") long userId,
+                                                                                @PathVariable(value = "bankAccountId") long bankAccountId) {
+
+        BankAccount bankAccount = bankAccountService.findBankAccountByUserId(userId);
```

Comment:
You'll never be able to accurately find bank account by user id since a user can have multiple accounts. Which one do you choose? You pass on the user id to ensure this belongs to the user but you need to say which bank account it to use.